xbuild/src/main.rs

```diff
@@ -29,6 +30,29 @@ enum Message {
 }
 
 fn main() {
+    let args: Vec<String> = env::args().collect();
+
+    if args.iter().any(|arg| arg == "--help" || arg == "-h") {
+        println!("xbuild - Build tool for xrizer OpenVR runtime");
+        println!("\nUsage: xbuild [OPTIONS] [CARGO_BUILD_ARGS]");
```

Comment:
```suggestion
        println!("\nUsage: xbuild [OPTIONS] -- [CARGO_BUILD_ARGS]");
```
I would recommend separating xbuild options from the `cargo build` options, otherwise it can lead to ambiguous parsing situations.